### PR TITLE
OKTA-433357 Updates to align with style guide

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Small tweaks to my recent EL merge to align it with the style guide.
- 
- **Is this PR related to a Monolith release?** No.

### Resolves:
* [OKTA-433357](https://oktainc.atlassian.net/browse/OKTA-433357)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-claims/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Create ID Token claims for OpenID Connect or access tokens for OAuth 2.0:
     * **Value** &mdash; this option appears if you choose **Expression**. Use Okta Expression Language syntax to generate values derived from attributes in Universal Directory and app profiles, for example: `appuser.username`.
     
       * See [Okta Expression Language](/docs/reference/okta-expression-language).
-      * See [Expressions for OAuth2/OIDC custom claims](/docs/reference/okta-expression-language/#expressions-for-oauth2-oidc-custom-claims) for custom claim-specific expressions.
+      * See [Expressions for OAuth 2.0/OIDC custom claims](docs/reference/okta-expression-language/#expressions-for-oauth-2-0-oidc-custom-claims) for custom claim-specific expressions.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-claims/index.md:18]: I was getting a 404 on this anchor on my local build, although it looks correct. Can you check?

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/add-custom-claim/index.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ To add a custom claim:
     * **Value** &mdash; this option appears if you choose **Expression**. Use Okta Expression Language syntax to generate values derived from attributes in Universal Directory and app profiles. For the ID token example, we use `user.preferredLanguage` and for the access token example, we use `user.secondEmail`. These reference an Okta user profile attribute.
     
       * See [Okta Expression Language](/docs/reference/okta-expression-language).
-      * See [Expressions for OAuth2/OIDC custom claims](/docs/reference/okta-expression-language/#expressions-for-oauth2-oidc-custom-claims) for custom claim-specific expressions.
+      * See [Expressions for OAuth 2.0/OIDC custom claims](docs/reference/okta-expression-language/#expressions-for-oauth-2-0-oidc-custom-claims) for custom claim-specific expressions.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/add-custom-claim/index.md:23]: same here. 404

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute/index.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: Include app-specific information in a custom claim
 ---
 
-If you want to include certain app-specific information in a token claim, you can do so by first adding the metadata to the profile section of the app. You can access any values that are put inside the app profile using `app.profile` written in Okta Expression Language. See [Expressions for OAuth2/OIDC custom claims](/docs/reference/okta-expression-language/#expressions-for-oauth2-oidc-custom-claims).
+If you want to include certain app-specific information in a token claim, you can do so by first adding the metadata to the profile section of the app. You can access any values that are put inside the app profile using `app.profile` written in Okta Expression Language. See [Expressions for OAuth 2.0/OIDC custom claims](docs/reference/okta-expression-language/#expressions-for-oauth-2-0-oidc-custom-claims).
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute/index.md:5]: This one, too.

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md [hunk 4]
```diff
@@ -386,7 +386,7 @@ See [Include app-specific information in a custom claim](/docs/guides/customize-
 
 #### Sample using access.scope
 
-In API Access Management custom authorization servers, you can name a claim `scope`. Then, you can use the expression `access.scope` to return an array of granted scope strings.
+In [API Access Management](docs/concepts/api-access-management/) custom authorization servers, you can name a claim `scope`. Then, you can use the expression `access.scope` to return an array of granted scope strings.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:389]: Also a 404 here ... you're missing the leading forward slash before docs (i.e., (/docs/concepts ...)

**Review by brianduffield-okta — CHANGES_REQUESTED** [review]
Line 33 needs an update (text before link) and line 58 needs same.

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-claims/index.md (hunk 1 above)
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-claims/index.md:18]:
```suggestion
      * See [Expressions for OAuth 2.0/OIDC custom claims](/docs/reference/okta-expression-language/#expressions-for-oauth-2-0-oidc-custom-claims) for custom claim-specific expressions.
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/add-custom-claim/index.md (hunk 2 above)
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/add-custom-claim/index.md:23]:
```suggestion
      * See [Expressions for OAuth 2.0/OIDC custom claims](/docs/reference/okta-expression-language/#expressions-for-oauth-2-0-oidc-custom-claims) for custom claim-specific expressions.
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute/index.md (hunk 3 above)
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute/index.md:5]:
```suggestion
If you want to include certain app-specific information in a token claim, you can do so by first adding the metadata to the profile section of the app. You can access any values that are put inside the app profile using `app.profile` written in Okta Expression Language. See [Expressions for OAuth 2.0/OIDC custom claims](/docs/reference/okta-expression-language/#expressions-for-oauth-2-0-oidc-custom-claims).
```

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md (hunk 4 above)
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:389]:
```suggestion
In [API Access Management](/docs/concepts/api-access-management/) custom authorization servers, you can name a claim `scope`. Then, you can use the expression `access.scope` to return an array of granted scope strings.
```

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md [hunk 5]
```diff
@@ -30,7 +30,7 @@ In addition to an Okta User Profile, all Users have a separate Application User
 | `$appuser.$attribute` | `$appuser` explicit reference to specific app<br>`$attribute` the attribute variable name  | zendesk.firstName<br>active_directory.managerUpn<br>google_apps.email |
 | `appuser.$attribute`  | `appuser` implicit reference to in-context app<br>`$attribute` the attribute variable name | appuser.firstName                                                     |
 
-> **Note:** Explicit references to apps aren't supported for OAuth/OIDC custom claims. See [Expressions for OAuth2/OIDC custom claims](/docs/reference/okta-expression-language/#expressions-for-oauth2-oidc-custom-claims).
+> **Note:** Explicit references to apps aren't supported for OAuth/OIDC custom claims. See [Expressions for OAuth 2.0/OIDC custom claims](docs/reference/okta-expression-language/#expressions-for-oauth-2-0-oidc-custom-claims).
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:33]:
```suggestion
> **Note:** Explicit references to apps aren't supported for OAuth/OIDC custom claims. See [Expressions for OAuth 2.0/OIDC custom claims](/docs/reference/okta-expression-language/#expressions-for-oauth-2-0-oidc-custom-claims).
```

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md (hunk 4 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md]: Fixed. Thanks for catching them :)

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md [hunk 6]
```diff
@@ -30,7 +30,7 @@ In addition to an Okta User Profile, all Users have a separate Application User
 | `$appuser.$attribute` | `$appuser` explicit reference to specific app<br>`$attribute` the attribute variable name  | zendesk.firstName<br>active_directory.managerUpn<br>google_apps.email |
 | `appuser.$attribute`  | `appuser` implicit reference to in-context app<br>`$attribute` the attribute variable name | appuser.firstName                                                     |
 
-> **Note:** Explicit references to apps aren't supported for OAuth/OIDC custom claims. See [Expressions for OAuth2/OIDC custom claims](/docs/reference/okta-expression-language/#expressions-for-oauth2-oidc-custom-claims).
+> **Note:** Explicit references to apps aren't supported for OAuth/OIDC custom claims. See [Expressions for OAuth 2.0/OIDC custom claims](/docs/reference/okta-expression-language/#expressions-for-oauth-2-0-oidc-custom-claims).
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:33]:
```suggestion
> **Note:** Explicit references to apps aren't supported for OAuth 2.0/OIDC custom claims. See [Expressions for OAuth 2.0/OIDC custom claims](/docs/reference/okta-expression-language/#expressions-for-oauth-2-0-oidc-custom-claims).
```

**Review by brianduffield-okta — APPROVED** [review]
All good! :shipit:

**Pull request merged**